### PR TITLE
OADP-932 install azure show bullets hvider

### PR DESCRIPTION
**Pull request by HagayVider1**
@kaovilai 
GH# : 
OCPBUGS : [OADP-932](https://issues.redhat.com//browse/OADP-932)

Showed hidden bullets in 'Prerequisites' section
https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html-single/backup_and_restore/index#migration-configuring-azure_installing-oadp-azure

Version : OADP 1.2.0
Issue : https://issues.redhat.com/browse/OADP-932

Version(s):

Issue:

Link to docs preview: https://59577--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by kaovilai — APPROVED** [review]

File: backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 :_content-type: ASSEMBLY
 [id="installing-oadp-mcg"]
-= Installing and configuring the OpenShift API for Data Protection with Multicloud Object Gateway
+= Installing and configuring the OpenShift API for Data Protection with OpenShift Data Foundation's Multicloud Object Gateway
```
Comment (amastbau) [inline backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc:3]: this doesn't seem to be related  to this PR @HagayVider1

Comment (anarnold97) [reply backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc:3]: @HagayVider1 - please could you fix this as Amos requested? Then add the necessary to get it moved to OCP review.

Thanks

**Review by HagayVider1 — COMMENTED** [review]
I think this was committed from a different ticket.
